.github/workflows/fast-dispatch-full-regressions-and-models.yaml

```diff
@@ -35,15 +112,15 @@ jobs:
       ARCH_NAME: ${{ matrix.test-group.arch }}
       LOGURU_LEVEL: INFO
       LD_LIBRARY_PATH: ${{ github.workspace }}/build/lib
-    runs-on: model-runner-${{ matrix.test-group.arch }}
+    runs-on: ${{ matrix.test-group.runs-on }}
```

Comment:
Are we removing the model-runner-*tags.
I'm not sure if there's anything special with those VMs with these tags